lib/chromedriver/helper.rb

```diff
@@ -35,6 +35,20 @@ def update
       download true
     end
 
+    def compatibility
+      if File.exists?("/usr/lib64/libstdc++.so.6")
+        if `strings /usr/lib64/libstdc++.so.6 | grep -e 'GLIBC.*3.4.15'`.empty?
+          chrome_dir = File.dirname(%x(readlink -f "`which google-chrome`").chomp)
+          path = "#{chrome_dir}:#{chrome_dir}/lib"
+          unless `echo $LD_LIBRARY_PATH`.empty?
```

Comment:
You should not exec in order to check presence/absence of an environment variable. You should use `ENV["LD_LIBRARY_PATH"]` in native Ruby.